docs/tutorials/tutorial-transform.md

```diff
@@ -0,0 +1,101 @@
+
+```sql
+INSERT INTO "transform_tutorial"
+WITH "ext" AS (
+  SELECT *
+  FROM TABLE(EXTERN('{"type":"inline","data":"{\"timestamp\":\"2018-01-01T07:01:35Z\",\"animal\":\"octopus\",  \"location\":1, \"number\":100}\n{\"timestamp\":\"2018-01-01T05:01:35Z\",\"animal\":\"mongoose\", \"location\":2,\"number\":200}\n{\"timestamp\":\"2018-01-01T06:01:35Z\",\"animal\":\"snake\", \"location\":3, \"number\":300}\n{\"timestamp\":\"2018-01-01T01:01:35Z\",\"animal\":\"lion\", \"location\":4, \"number\":300}"}', '{"type":"json"}')) EXTEND ("timestamp" VARCHAR, "animal" VARCHAR, "location" BIGINT, "number" BIGINT)
+)
+SELECT
+  TIME_PARSE("timestamp") AS "__time",
+  TEXTCAT('super-', "animal") AS "animal",
+  "location",
+  "number",
+  "number" * 3 AS "triple-number"
+FROM "ext"
+WHERE (TEXTCAT('super-', "animal") = 'super-mongoose' OR "location" = 3 OR "number" * 3 = 300)
+PARTITIONED BY DAY
+```
+
+The following columns have transformations applied on ingestion:
+* `animal`: prepends "super-" to the values in the `animal` column using the [`TEXTCAT`](../querying/sql-functions.md/#textcat) function. This will override the `animal` column with the transformed version.
+* `triple-number`: multiplies the `number` column by 3. This stores the results in a new column called `triple-number`. Note that it ingestions both the original and the transformed column.
+
+Additionally, a filter with the following three OR clauses are applied:
+* `animal` values that match "super-mongoose"
+* `triple-number` values that match 300
+* `location` values that match 3
+
+## Query the transformed data
+
+In the web console, open a new tab in the **Query** view. Run the following query to view the ingested data:
+
+```sql
+SELECT * FROM "transform_tutorial"
+```
+
+Returns the following:
+
+| `__time` | `animal` | `location` | `number` | `triple-number` | 
+| -- | -- | -- | -- | -- |
+| `2018-01-01T05:01:35.000Z` | `super-mongoose` | `2` | `200` | `600` |
+| `2018-01-01T06:01:35.000Z` | `super-snake` | `3` | `300` | `900` |
+| `2018-01-01T07:01:35.000Z` | `super-octopus` | `1` |  `100` | `300` |
+
+Note that the filter applies after the transformations. The only row that isn't ingested from the input data is the "lion" row because it doesn't meet any of the three filter condition listed. 
+
+## Learn more
+
+See the following topics for more information:
+
+* [All functions](../querying/sql-functions.md) for a list of functions that can be used to transform data. 
+* [Transform spec reference](../ingestion/ingestion-spec.md/#transformspec) to learn more about transforms in JSON-based batch ingestion.
+* [Query filters](../querying/filters.md) to learn about the types of filters available. 
+* [WHERE clause](../querying/sql.md#where) to learn how to create filters in SQL.
```

Comment:
Possibly true for this one too, but it's more related in that you do use WHERE to add filters.

Reply:
I choose to keep this one since it shows two separate examples of `WHERE` clause, and some of the nuance for it, like it can reference a subquery. I will keep it unless it is decided to remove the filter aspect of the tutorial